data/korath ships.txt

```diff
@@ -146,6 +145,7 @@ ship "Korath World-Ship"
 		"outfit space" 778
```

Comment:
Same as the Raider.

Reply:
+2 outfits, -2 mass, to get 20 free space? (See Raider reply)